### DISCUSSION TITLE
Issue with oracle

### DISCUSSION THREAD

**Discussion by Dabeto83** [grate-devs/grate#305] [General]
Hi...

I'm trying to create a basic table against oracle xe 21c. The table is created but I got the issue:

Unhandled exception: grate.Exceptions.MigrationFailed: Migration failed due to errors:
 * ORA-00922: missing or invalid option
[0001_CreateTables.txt](https://github.com/erikbra/grate/files/10736840/0001_CreateTables.txt)

 ---> Oracle.ManagedDataAccess.Client.OracleException (0x80004005): ORA-00922: missing or invalid option
 
 Could you please confirm if I'm doing something wrong?
 
 I've executed the same script in oracle sql developer and it works fine.

**Comment by wokket** [comment]
This may be an instance of #283 ?  If you post your script, or check for semicolons we may be able to help some more.
